### DISCUSSION TITLE
pdf not showing some unicode correctly

### DISCUSSION THREAD

**Discussion by Lightup1** [quarto-dev/quarto-cli#1626] [Q&A] [answered]
Example:
```
---
title: error showing unicode
author: Lightup
date: \today
format:
  pdf: 
    include-in-header: 
      text: |
        \usepackage[utf8x]{inputenc}
jupyter: julia-1.8
---
ħ is showing,but ω is not showing.
Also for code cell
```

![image](https://user-images.githubusercontent.com/32999017/182155817-e696575d-ca5e-4c91-8528-1cc5d13fd54b.png)

**Comment by dragonstyle** [comment, marked as answer]
I think that the issue can often be the the font used to render the PDF doesn't support some unicode characters. If this is the issue - this comment seems likely to be helpful:

https://github.com/quarto-dev/quarto-cli/issues/1559#issuecomment-1199296348

HTH!

**Reply by Lightup1** [reply]
Thanks, the issue is truely caused by the font specification.
I just fix that follow that comment.
